### PR TITLE
Prepare release v1.0.4

### PR DESCRIPTION
**Pull request by timoreimann**

### CODE REVIEW

File: deploy/kubernetes/releases/csi-digitalocean-v1.0.4.yaml [hunk 1]
```diff
@@ -0,0 +1,526 @@
+# Copyright 2018 DigitalOcean
```
Comment (jcodybaker) [inline deploy/kubernetes/releases/csi-digitalocean-v1.0.4.yaml]: Should we bump this copyright date in the template?

Comment (timoreimann) [reply deploy/kubernetes/releases/csi-digitalocean-v1.0.4.yaml]: Yeah, I guess we should do that. How about doing one big update in a dedicated, follow-up PR though? Or would you prefer updating just the template in this one?

Comment (jcodybaker) [reply deploy/kubernetes/releases/csi-digitalocean-v1.0.4.yaml]: Yeah, I don't have strong feelings on it.  It's open source software; worst case its copyright expires and it falls into the public domain.

Comment (timoreimann) [reply deploy/kubernetes/releases/csi-digitalocean-v1.0.4.yaml]: Done in #272.

**Pull request merged**